src/main/java/frc/robot/lib/interfaces/Arm.java

```diff
@@ -141,18 +156,12 @@ public double applyDeadband(double armManualInput) {
         return 0.0;
     }
 
-    public double getScoringDirectionModifier() {
```

Comment:
We do not know if moving the arm out the back of the robot is totally out of the question yet. Keep these functions in for now.